test/controllers/MakePaymentControllerSpec.scala

```diff
@@ -286,6 +286,27 @@ class MakePaymentControllerSpec extends ControllerBaseSpec {
         status(result) shouldBe Status.FORBIDDEN
       }
     }
+
+    "the user is insolvent and not continuing to trade" should {
+
+      "return 403 (Forbidden)" in new MakePaymentDetailsTest {
+        override val authResult: Future[~[Enrolments, Option[AffinityGroup]]] = successfulAuthResult
```

Comment:
No need to override `authResult` as it's already set to `successfulAuthResult` by default in the trait